README.md

```diff
@@ -32,3 +32,14 @@ By participating in this project, you agree to abide by our [code of conduct](./
 
 [help wanted]: https://github.com/mozilla/universal-search/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22
 [mailing list]: https://mail.mozilla.org/listinfo/testpilot-dev
+
+## Coding conventions
+
+- lib/universal-search.js loads code into windows and constructs the object graph
+- use constructors to assign dependencies to the object, and bind `this` to callbacks
+- use `init` for initialization work other than dependency injection
+- use `destroy` to unset anything that could leak memory
+- leading underscore in prototype methods indicates private API
+- event naming convention: hyphenated lower case.
+	- we don't need to namespace it to individual items because it's such a small list of events. just keep it simple.
```

Comment:
This is fantastic, and exactly the sort of thing that will make my life easier. Thanks so much for this.